src/app/src/main/kotlin/org/imperial/mrc/hint/clients/ADRFuelClient.kt

```diff
@@ -7,6 +7,7 @@ import org.imperial.mrc.hint.logging.GenericLogger
 import org.imperial.mrc.hint.logging.logADRRequestDuration
 import org.imperial.mrc.hint.security.Encryption
 import org.imperial.mrc.hint.security.Session
+import org.imperial.mrc.hint.security.oauth2.ProfileDefinition.Companion.token
```

Comment:
Just a question for my understanding, how does this token get set when importing like this? From reading about companion objects it sounds like this will get this like a static value but that looks to be an empty string. https://github.com/mrc-ide/hint/blob/5903eecaf25ed70c10b7b3c276785aa1afb5d0f9/src/app/src/main/kotlin/org/imperial/mrc/hint/security/oauth2/ProfileDefinition.kt#L21 Is kotlin doing something clever here?